tokio/src/io/driver/interest.rs

```diff
@@ -1,38 +1,89 @@
+use crate::io::Ready;
+
 use std::fmt;
 use std::ops;
 
 /// Readiness event interest
 ///
 /// Specifies the readiness events the caller is interested in when awaiting on
 /// I/O resource readiness states.
-#[derive(Clone, Copy)]
-pub(crate) struct Interest(mio::Interest);
+#[derive(Clone, Copy, Eq, PartialEq)]
+pub struct Interest(mio::Interest);
```

Comment:
The `Interest` struct itself has no feature flag requirements at all (at least not in the doc), but all of its methods and constants have the following note:

> This is supported on crate feature net, or crate feature process, or Unix and crate feature signal only.

Can we simplify the feature flag situation?

The same applies to the `Ready` struct.

Reply:
I fixed, it now only shows `net` on the struct level and nothing on the methods.